cttportfolio.css

```diff
@@ -1,470 +1,874 @@
-nav{
-	display:flex;
-	padding:0 5%;
-	justify-content:space-between;
-	align-items:center;
-	position:fixed;
-	top:0;
-	width:100vw;
-	z-index:1;
-	margin:10px 0;
-}
-nav a, nav p{
-	text-decoration:none;
-	color:rgb(248,248,255);
-	text-transform:uppercase;
-}
-.navlogo > img{
-	padding-right:15px;
-	height:100%;
-	filter: invert(0.95); 
-	position:relative;
-	bottom:4px;
-	box-shadow:none;
-}
-
-.navlogo > p{
-	font-size:2.5rem;
-	font-weight:700;
-}
-.navlogo{
-	display:flex;
-	align-items:center;
-	height:80px;
-}
-.navlinks > a{
-	font-size:2rem;
-	margin-left:30px;
-	padding:15px 10px;
-	padding-top:18px;
-	border:none;
+
+nav {
+	display: flex;
+	padding: 0 5%;
+	justify-content: space-between;
+	align-items: center;
+	position: fixed;
+	top: 0;
+	width: 100vw;
+	z-index: 1;
+	margin: 10px 0;
+}
+
+nav a,
+nav p {
+	text-decoration: none;
+	color: rgb(248, 248, 255);
+	text-transform: uppercase;
+}
+
+.navlogo>img {
+	padding-right: 15px;
+	height: 100%;
+	filter: invert(0.95);
+	position: relative;
+	bottom: 4px;
+	box-shadow: none;
+}
+
+.navlogo>p {
+	font-size: 2.5rem;
+	font-weight: 700;
+}
+
+.navlogo {
+	display: flex;
+	align-items: center;
+	height: 80px;
+}
+
+.navlinks>a {
+	font-size: 2rem;
+	margin-left: 30px;
+	padding: 15px 10px;
+	padding-top: 18px;
+	border: none;
 	border-radius: 5px;
-	font-weight:600;
+	font-weight: 600;
+}
+
+.navlinks a:hover {
+	border: 1px solid rgb(248, 248, 255);
 }
-.navlinks a:hover{
-	border:1px solid rgb(248,248,255);
 
+.nav_dropdown {
+	display: none;
 }
+
+
 /*======================================== HEADER SECTION ========================================*/
-header{
-	width:100vw;
-	height:100vh;
-	position:relative;
-	z-index:-1;
-}
-#frontpagevideo{
-	object-fit:cover;
-	width:inherit;
-	max-height:inherit;
-	position:fixed;
-}
-.headertext{
-	position:absolute;
-	top:0;
-	left:0;
-	width:inherit;
-	height:inherit;
-	display:flex;
-	flex-direction:column;
-	justify-content:center;
-	padding-left:20%;
-}
-h1{
-	font-size:2.5rem;
-	color:white;
-}
-h2{
-	font-size:1.8rem;
-	text-transform:uppercase;
-	color: rgb(248,248,255);
+
+header {
+	width: 100vw;
+	height: 100vh;
+	position: relative;
+	z-index: -1;
 }
+
+#frontpagevideo,
+#frontpagevideo[poster] {
+	object-fit: cover;
+	max-width: 100%;
+	position: fixed;
+}
+
+.headertext {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: inherit;
+	height: inherit;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	padding-left: 20%;
+}
+
+h1 {
+	font-size: 2.5rem;
+	color: white;
+}
+
+h2 {
+	font-size: 1.8rem;
+	text-transform: uppercase;
+	color: rgb(248, 248, 255);
+}
+
+
 /*======================================== ABOUT ME SECTION ========================================*/
-.aboutme{
-	height:1200px;
-	width:100%;
-	display:flex;
-	flex-direction:column;
-	justify-content:space-between;
+
+.aboutme {
+	height: 1200px;
+	width: 100%;
```

Comment:
Elements which have a `display` type of `block` have a `width` of `100%` by default, therefore this does nothing.